src/prefect/events/schemas/automations.py

```diff
@@ -427,6 +430,49 @@ class AutomationCore(PrefectBaseModel, extra="ignore"):
         default=None, description="The owning resource of this automation"
     )
 
+    @classmethod
+    @sync_compatible
+    async def create(cls: Type[Self], automation: Type[Self]) -> UUID:
+        client, _ = get_or_create_client()
+        automation = await client.create_automation(automation=automation)
+        return automation if automation else None
+
+    @classmethod
+    @sync_compatible
+    async def update(cls: Type[Self], id: UUID, automation: Type[Self]) -> UUID:
+        client, _ = get_or_create_client()
+        print(automation)
+        automation = await client.update_automation(id=id, automation=automation)
+        return automation if automation else None
```

Comment:
`type[T]` is for when the thing is a literal `type`, not an instance of the type, where `*_automation` methods accept an instance of `AutomationCore`
```suggestion
    async def create(cls: Type[Self], automation: Self) -> UUID:
        client, _ = get_or_create_client()
        automation = await client.create_automation(automation=automation)
        return automation if automation else None

    @classmethod
    @sync_compatible
    async def update(cls: Type[Self], id: UUID, automation: Self) -> UUID:
        client, _ = get_or_create_client()
        automation = await client.update_automation(id=id, automation=automation)
        return automation if automation else None
```

Reply:
also I got distracted by the type hinting, but its interesting that these are classmethods but don't need to use the `cls` at all

was the choice to have these be class methods DX focused?